src/compas_fab/robots/trajectory.py

```diff
@@ -21,17 +22,19 @@ class JointTrajectoryPoint(Configuration):
     Trajectory points are defined either as *values + velocities and
     accelerations*, or as *values + efforts*.
 
-    Attributes
+    Parameters
```

Comment:
What is the criteria for choosing `Parameters` over `Attributes`? According to the [this doc](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) it seems to be recommending having BOTH! 😱

Reply:
[Numpydocs](https://numpydoc.readthedocs.io/en/latest/format.html#id10) agrees..

Maybe we can think of Parameters as documentation for __init__ only, and then document the attributes for the class (including things straight from __init__)?

Reply:
The "canonical" example for how to use and which style to use is still this document: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

> Maybe we can think of Parameters as documentation for init only, and then document the attributes for the class (including things straight from init)?

Yes, that is correct. This is explained in the sphinx document linked above:

    The __init__ method may be documented in either the class level
    docstring, or as a docstring on the __init__ method itself.

    Either form is acceptable, but the two should not be mixed. Choose one
    convention to document the __init__ method and be consistent with it.

We always use the former option of documenting it at the class level.

Reply:
34f6156